requirements.txt

```diff
@@ -1,7 +1,20 @@
+alembic
+beautifulsoup4
+bunch
 Flask
+Flask-WTF
+itsdangerous
+Jinja2
```

Comment:
No need to include this one as it will come up with flask directly, same for ``itsdangerous`` iirc